Source/miniwin/misc_msg.cpp

```diff
@@ -311,7 +315,22 @@ bool FetchMessage_Real(tagMSG *lpMsg)
 		return true;
 	}
 
-#ifndef USE_SDL1
+#if !defined(USE_SDL1) && !defined(__vita__)
+	if (!movie_playing) {
+		if (IsAnyOf(e.type, SDL_MOUSEBUTTONDOWN, SDL_MOUSEBUTTONUP) && e.button.which == SDL_TOUCH_MOUSEID)
+			return true;
+		if (e.type == SDL_MOUSEMOTION && e.motion.which == SDL_TOUCH_MOUSEID)
+			return true;
+		if (e.type == SDL_MOUSEWHEEL && e.wheel.which == SDL_TOUCH_MOUSEID)
+			return true;
+	}
+#endif

```

Comment:
I'm a little unsure what this is handeling, could you add a comment for it?

My reading is that it ignores all touch input, but then I don't understand what the `HandleTouchEvent(e);` below is supposed to do ...

Reply:
Thanks, I forgot about the build in virtual mouse events in SDL :)